frontend/riskofbias/constants.js

```diff
@@ -80,15 +80,20 @@ const NA_KEYS = [10, 20],
     BIAS_DIRECTION_UNKNOWN = 0,
     BIAS_DIRECTION_UP = 1,
     BIAS_DIRECTION_DOWN = 2,
+    BIAS_DIRECTION_COMPACT = {
+        [BIAS_DIRECTION_UNKNOWN]: "-",
+        [BIAS_DIRECTION_UP]: "⯅",
+        [BIAS_DIRECTION_DOWN]: "⯆",
+    },
     BIAS_DIRECTION_SIMPLE = {
         [BIAS_DIRECTION_UNKNOWN]: "?",
-        [BIAS_DIRECTION_UP]: "⬆",
-        [BIAS_DIRECTION_DOWN]: "⬇",
+        [BIAS_DIRECTION_UP]: "🡩",
```

Comment:
these don't render for me; I think they're not defined in all fonts. I think we'll need to revert back to the prior arrows for compatibility

<img width="130" alt="Screen Shot 2020-04-23 at 10 33 03 AM" src="https://user-images.githubusercontent.com/999952/80111491-f35a9f80-854d-11ea-9536-ac33615fb5e3.png">